docs/integrations/tax.mdx

```diff
@@ -83,6 +83,12 @@ Make sure to also review our [app development best practices](/docs/integrations
   When you develop your app, you can register your app in the Developer Portal without submitting it. The app will be in a Draft state. This means only stores owned by the same email address as that on the Developer Portal account can install the app.
 </Callout>
 
+### Building resilience
+
+We encourage you to write robust, resilient code that will not break end to end functionality when BigCommerce pushes a non-breaking change to the Tax Provider API (e.g., adding a new request field).
+
+See our [API best practices](/docs/start/best-practices) for more detailed guidance.
```

Comment:
I wonder if we should state here `for more detailed guidance on what BC considers a non breaking change`.

I say this as the previous provider (and vertex) thought any new field was a `breaking` change.